similar/main/state.cpp

```diff
@@ -922,7 +922,12 @@ static d_game_unique_state::save_slot state_get_savegame_filename(d_game_unique_
 		const auto uchoice = static_cast<std::size_t>(choice);
 		fname = filename[uchoice];
 		if (dsc)
+		{
 			*dsc = desc[uchoice];
+			if (!dsc->data()[0]) {
+				strcpy(dsc->data(), "-no title-");
```

Comment:
It might be more useful to generate a descriptive string here.  For autosaves, I insert _month-day hour:minute:second_.

https://github.com/dxx-rebirth/dxx-rebirth/blob/a20551191afc87b4fc7e2afdeb97c254458f0984/similar/main/state.cpp#L691-L695

You probably don't want the literal text `auto` for the case of a joystick-entered save.  Note that the maximum description length prevents using a verbose description.

I can handle changing this later.

Reply:
I agree that it would be a neat feature, but then we'd also need some kind of tracking or detection that a save slot has such an auto-generated name and update the timestamp when saving to the same slot later on. Otherwise, the slot would always keep the timestamp from when it was first created, which is not _that_ more helpful than the current "no title" ;)